docs/workflows/cumulus-task-message-flow.md

```diff
@@ -13,15 +13,17 @@ The steps in this flow are detailed in sections below.
 
 ## Cumulus Message Format
 
-Cumulus Messages come in 2 flavors: The full **Cumulus Message** and the **Cumulus Remote Message**. The Cumulus Remote Message points to a full Cumulus Message stored in S3 because of size limitations.
+A full **Cumulus Message** has the following keys:
```

Comment:
Given the contextual concerns of defining a "Cumulus Message" given we've already added keys in the current codebase, it feels better to define it in terms of which root keys are reserved/have meaning and remove the distinction between a remote/non-remote message given it's not a entire-message replacement.